src/gui/render.cpp

```diff
@@ -1145,7 +1145,7 @@ static void init_render_settings(Section_prop& secprop)
 	        "                 contrast.\n"
 	        "You can also set custom colours by specifying 16 space or comma separated\n"
 	        "colour values, either as 3 or 6-digit hex codes (e.g. #f00 or #ff0000 for full\n"
-	        "red), or decimal RGB triplets (e.g. (255, 0, 255) for magenta). The 16 colours\n"
+	        "red), or decimal RGB triplets (e.g. '(255, 0, 255)' for magenta). The 16 colours\n"
```

Comment:
Hmm, maybe remove the quotes here? I did not use quotes for the hex values.

Reply:
True. I added them because of the double parenthesis, but they are not necessary

Reply:
People won't be confuse by the double parenthesis, but could be confused by the quotes. So I'd go with the least confusing option :)